cmd/flags.go

```diff
@@ -332,7 +332,7 @@ func dstPortFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 }
 
 func debugServerFlags(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
-	cmd.Flags().String(flagDebugAddr, defaultDebugAddr, "address to use for debug server. Set empty to disable debug server.")
+	cmd.Flags().String(flagDebugAddr, "", "address to use for debug server. Set empty to disable debug server.")
```

Comment:
With the default behavior changing I think the description here could be improved. Maybe referencing the config field that is being consumed now or something.